src/features/bridge/templates/BottomSheetBridgePreview/components/GeneralPreviewTemplate/GeneralPreviewTemplate.tsx

```
@@ -0,0 +1,123 @@
+import { useBridgeContextData } from '@features/bridge/context';
+import { BridgeTransactionHistoryDTO } from '@models/dtos/Bridge';
+import { Row, Spacer, Spinner, Text } from '@components/base';
+import { TouchableOpacity, View } from 'react-native';
+import { COLORS } from '@constants/colors';
+import { useTranslation } from 'react-i18next';
+import { NumberUtils } from '@utils/number';
+import { scale } from '@utils/scaling';
+import { CloseCircleIcon } from '@components/svg/icons/v2';
+import { PrimaryButton } from '@components/modular';
+
+interface GeneralPreviewDataModel {
+  errorBalance: boolean;
+  onAcceptPress: () => Promise<void>;
+  onClose: () => void;
+}
+
+export const GeneralPreviewTemplate = ({
+  errorBalance,
+  onAcceptPress,
+  onClose
+}: GeneralPreviewDataModel) => {
+  const { t } = useTranslation();
+
+  const { methods, variables } = useBridgeContextData();
+  const { setProcessingTransaction } = methods;
+  const {
+    processingTransaction,
+    amountToBridge,
+    selectedTokenFrom,
+    destinationData
+  } = variables;
+
+  const header = t('bridge.transaction.preview.header')
+    .replace(
+      '{{amount}}',
+      NumberUtils.limitDecimalCount(amountToBridge || '', 2)
+    )
+    .replace('{{symbol}}', selectedTokenFrom.symbol || '')
+    .replace('{{network}}', destinationData.value.name);
+
+  useEffect(() => {
+    if (!processingTransaction?.withdrawTx && processingTransaction) {
+      const getTransaction = async (
+        _transaction: BridgeTransactionHistoryDTO
+      ) => _transaction.wait();
+      try {
+        getTransaction(processingTransaction).then((transaction) => {
+          setProcessingTransaction({
+            ...processingTransaction,
+            eventId: 0,
+            userTo: transaction.to,
+            withdrawTx: transaction.transactionHash
+          });
+          return transaction;
+        });
+      } catch (e) {
+        // ignore
```

do not ignore error block, throw error